### ISSUE TITLE
The API team pushed breaking changes

### ISSUE THREAD

**Issue by mustberuss** [mikeym88/PatentsView-API-Wrapper#4]
The API team made changes to the new version of the API that broke the wrapper.  They changed the url of the patent_citation endpoint, changed the way that paging works and some of attribute names changed.   I got it to work in my fork and could submit a PR if you'd like.  It's about a dozen commits, with most of the changes in main.py

**Comment by mikeym88** [comment]
Sure! Please submit the PR and will review it. Do you have a link to the change that the API team made?

**Comment by mustberuss** [comment]
PR #5 submitted.  There isn't a page that documents the most recent set of changes.  The closest thing is the  [Swagger UI page](https://search.patentsview.org/swagger-ui/) that shows the new and moved endpoints (some are now nested under /patent ) and the new field names.

I'm also working on updates to the R package and the latest round of API changes broke some of the test cases.  One odd change renames patent_number to patent_id.

**Comment by mustberuss** [comment]
I did find [this page](https://patentsview.org/api-v01-information-page) explaining the overall API change, some of it was addressed in #3.  It says to refer to the 200 responses of the Swagger UI page to see the fields that are returned- their endpoint documentation is lagging.  At the bottom of the page it says they will make incremental releases so the wrapper may break again before the API finalized!
